lambda-otel4s/shared/src/main/scala/feral/lambda/otel4s/FaasAttributes.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda.otel4s
+
+import feral.lambda.Context
+import org.typelevel.otel4s.Attribute
+import org.typelevel.otel4s.semconv.resource.attributes.ResourceAttributes
+import org.typelevel.otel4s.semconv.trace.attributes.SemanticAttributes
+
+object LambdaContextTraceAttributes {
```

Comment:
Since semantic conventions 1.24.0, attributes are split into stable and experimental.

Stable attributes are available in the `otel4s-semconv` package and experimental in the `otel4s-semconv-experimental`. 

Also, attributes aggregated by the namespaces, for example, [FaasExperimentalAttributes](https://github.com/typelevel/otel4s/blob/main/semconv/experimental/src/main/scala/org/typelevel/otel4s/semconv/experimental/FaasExperimentalAttributes.scala).

Experimental attributes may have binary incompatible changes between releases, so libraries should not use them directly. 

____

Can feral pull the `otel4s-semconv-experimental`?

Reply:
Whether libraries should pull that in is an interesting question, but it appears all of the FAAS/Cloud attributes are in experimental, so I've added it. Otherwise there aren't really any attributes to add

Reply:
Options:
1. Add `experimental`
2. Remove experimental attributes and wait for them to be stable
3. Re-use attribute keys to create new attributes inline, and port to experimental later